src/segmenter/latin.rs

```diff
@@ -0,0 +1,33 @@
+use unicode_segmentation::UnicodeSegmentation;
+
+use super::Segmenter;
+
+/// Latin specialized [`Segmenter`].
+///
+/// This Segmenter uses [`UnicodeSegmentation`] internally to segment the provided text.
+pub struct LatinSegmenter;
+
+impl Segmenter for LatinSegmenter {
+    fn segment_str<'o>(&self, s: &'o str) -> Box<dyn Iterator<Item = &'o str> + 'o> {
+        Box::new(s.split_word_bounds())
```

Comment:
We will see a PR after this one to split by `'`? https://github.com/meilisearch/charabia/issues/90

Reply:
Yup, this PR is already too big.